### ISSUE TITLE
Cannot update template internal names

### ISSUE THREAD

**Issue by g-psantos** [chris2511/xca#444]
I am setting up a set of templates for various types of CA and leaf certificates. Since templates build off of one another, I generally duplicate an existing template and then update its properties and internal name. However, the updated internal name is _not_ saved when I close the database and later reopen it. Comments are also not updated, though Key Usage and Extensions seem to be.

Running XCA 2.4.0 Portable on Windows 11.

**Comment by itsKV** [comment]
I can reproduce the bug. 
+1

**Comment by chris2511** [comment]
Works with current tip of development and with the next release.

**Comment by itsKV** [comment]
Do I have to compile the source for myself? I could see the final release of 2 years old.

**Comment by AnonATXDO** [comment]
> Do I have to compile the source for myself? I could see the final release of 2 years old.

See under https://github.com/chris2511/xca/actions/workflows/cmake.yaml

**Comment by g-psantos** [comment]
I couldn't verify the fix on Windows, as the latest build of XCA for Windows outright crashes when duplicating a template.

I downloaded the Windows artifacts from the [latest CI run](https://github.com/chris2511/xca/actions/runs/6241477717). I tried with both Qt 5 and 6 and used the portable archive. The crash occurs with a new database as well as an existing one.

Should I file the crash as a separate issue?
